src/components/VmStatus/VmStatus.js

```diff
@@ -50,8 +50,13 @@ const StateImporting = ({ additionalText, ...props }) => (
 );
 
 const StateV2VConversionInProgress = ({ progress, ...props }) => (
-  <LinkStatus icon="import" {...props}>
-    V2V Conversion In Progress
+  <LinkStatus
+    icon="import"
+    title="Importing (VMWare)"
+    information="Virtual Machine is being converted from VMware format to libvirt format."
```

Comment:
btw I think these messages could be passed from `vmStatus.js` and changed accordingly there (can be customized if the message is parsed from pod, etc.)